resources/views/frontend/layouts/partials/colnav.blade.php

```diff
@@ -0,0 +1,47 @@
+<div class="contentwrap">
+  <div class="colnav">    
+    <div id="firstlogin" class="box box_login ">
+      <div class="bg_t">
+      </div>
+      <div class="content">
+        <img src="frontend/imgs/quatang.jpg" alt="">
```

Comment:
[rank C] why are you alway using hardcode paths? Please check and make it as Laravel path function (url ())